infrastructure/service-bus.tf

```diff
@@ -5,3 +5,9 @@ module "sscs-hearings-event-queue" {
   resource_group_name = "sscs-${var.env}"
   requires_session    = true
 }
+
+resource "azurerm_key_vault_secret" "servicebus_primary_shared_access_key" {
+  name         = "tribunals-servicebus-shared-access-key"

```

Comment:
I'd expand this name to be be something like tribunals-hearing-queue-shared-access-key

Reply:
Yup that sounds alot better updated the name!